packages/storage-compat/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@firebase/storage-compat",
+  "version": "0.1.0",
+  "description": "The Firebase Firestore compatibility package",
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "main": "./dist/index.cjs.js",
+  "browser": "./dist/index.esm2017.js",
+  "module": "./dist/index.esm2017.js",
+  "esm5": "./dist/index.esm5.js",
+  "license": "Apache-2.0",
+  "typings": "./dist/src/index.d.ts",
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "build": "rollup -c rollup.config.js && yarn add-compat-overloads",
+    "build:deps": "lerna run --scope @firebase/storage-compat --include-dependencies build",
+    "dev": "rollup -c -w",
+    "test": "run-p test:browser test:node lint",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:browser test:node",
+    "test:browser:unit": "karma start --single-run --unit",
+    "test:browser:integration": "karma start --single-run --integration",
+    "test:browser": "karma start --single-run",
+    "test:node": "TS_NODE_FILES=true TS_NODE_CACHE=NO TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --reporter lcovonly -- mocha 'test/{,!(browser)/**/}*.test.ts' --file src/index.ts --config ../../config/mocharc.node.js",
+    "test:debug": "karma start --browser=Chrome",
+    "prettier": "prettier --write 'src/**/*.ts' 'test/**/*.ts'",
+    "add-compat-overloads": "ts-node-script ../../scripts/exp/create-overloads.ts -i ../storage/dist/storage-public.d.ts -o dist/src/index.d.ts -a -r FirebaseStorage:types.FirebaseStorage -r StorageReference:types.Reference -r FirebaseApp:FirebaseAppCompat --moduleToEnhance @firebase/storage"
+  },
+  "peerDependencies": {
+    "@firebase/app-compat": "0.x"
+  },
+  "dependencies": {
+    "@firebase/storage": "0.6.1",
```

Comment:
Make sure this matches `packages/storage/pacakge.json` (currently 0.6.2, will change again Thursday 8/19)